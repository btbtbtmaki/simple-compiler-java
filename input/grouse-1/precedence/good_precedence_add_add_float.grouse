main {

  print 5.0 + 6.4 + 9.6 nl;
  print 6.8 + 4.9 + 3.0 nl;
  print 8.6 + 1.9 + 9.1 nl;
  print 7.7 + 9.3 + 9.4 nl;
  print 4.4 + 2.8 + 5.4 nl;
  print 5.1 + 8.6 + 5.7 nl;
  print 5.0 + 7.3 + 1.9 nl;
  print 3.5 + 9.4 + 7.4 nl;
  print 3.5 + 4.0 + 5.2 nl;
  print 7.3 + 2.9 + 8.6 nl;
  print 8.9 + 8.0 + 6.7 nl;
  print 8.6 + 7.6 + 4.7 nl;

  print 8.7 + 4.4 + 5.0 nl;
  print 7.7 + 7.5 + 5.5 nl;
  print 2.7 + 8.2 + 6.9 nl;
  print 6.8 + 7.7 + 9.8 nl;
  print 9.2 + 7.6 + 2.3 nl;
  print 6.1 + 2.4 + 4.4 nl;
  print 5.2 + 9.9 + 6.8 nl;
  print 9.7 + 1.5 + 1.3 nl;
  print 3.2 + 6.1 + 5.4 nl;
  print 4.7 + 5.5 + 1.0 nl;
  print 8.3 + 5.4 + 2.0 nl;
  print 7.2 + 4.4 + 6.4 nl;

  print 9.4 + 8.1 + 9.7 + 8.9 nl;
  print 2.4 + 7.2 + 7.7 + 3.8 nl;
  print 3.8 + 3.1 + 1.9 + 3.4 nl;
  print 1.0 + 3.0 + 2.7 + 1.3 nl;
  print 2.5 + 7.5 + 5.5 + 1.6 nl;
  print 9.0 + 5.2 + 3.5 + 2.4 nl;
  print 2.1 + 2.4 + 3.6 + 7.5 nl;
  print 9.0 + 4.3 + 9.2 + 4.0 nl;
  print 7.6 + 7.2 + 8.0 + 7.8 nl;
  print 8.6 + 4.2 + 6.6 + 3.1 nl;
  print 1.5 + 6.0 + 9.0 + 2.4 nl;
  print 1.8 + 2.2 + 8.0 + 2.3 nl;

  print 8.1 + 4.9 + 5.5 + 8.3 nl;
  print 3.4 + 9.9 + 7.7 + 8.3 nl;
  print 7.9 + 1.0 + 1.0 + 9.2 nl;
  print 6.5 + 8.8 + 5.4 + 5.9 nl;
  print 7.4 + 7.5 + 3.2 + 8.7 nl;
  print 6.3 + 8.9 + 9.1 + 6.1 nl;
  print 5.0 + 1.0 + 2.1 + 7.0 nl;
  print 8.5 + 1.8 + 4.8 + 5.4 nl;
  print 9.8 + 4.3 + 1.4 + 4.8 nl;
  print 2.3 + 2.5 + 1.2 + 7.7 nl;
  print 8.0 + 9.3 + 6.7 + 4.3 nl;
  print 5.0 + 8.5 + 9.7 + 4.3 nl;

}

