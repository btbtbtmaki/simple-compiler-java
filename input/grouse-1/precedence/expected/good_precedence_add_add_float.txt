21
14.7
19.6
26.4
12.6
19.4
14.2
20.3
12.7
18.8
23.6
20.9
18.1
20.7
17.8
24.3
19.1
12.9
21.9
12.5
14.7
11.2
15.7
18
36.1
21.1
12.2
8
17.1
20.1
15.6
26.5
30.6
22.5
18.9
14.3
26.8
29.3
19.1
26.6
26.8
30.4
15.1
20.5
20.3
13.7
28.3
27.5
